### DISCUSSION TITLE
Requesting Additional Maintainers

### DISCUSSION THREAD

**Discussion by deankarn** [go-playground/validator#1330] [General]
Hey all 👋 

It's been a pleasure maintaining this package for nearly a decade, with the help of the community and some special people, and seeing all the value it brings to the community ❤️ 

This project though has grown large enough that more than a few people are required to properly support the community. I unfortunately don't have the same free time that I once did, have spread myself maybe too thin maintaining multiple open source projects in Go, Rust, etc.. and am unable to keep up with the influx of questions, PR's and issues,, but want this package to continue to grow and thrive for everyone!

To that end this is a call for maintainers to help ensure the community is supported in a proper and timely manner!

To set some expectation before throwing your hat in the ring, because it's not all sunshine and roses, as they say:

- Backward compatibility and breaking changes **MUST** be taken extremely seriously, especially since this package is used to prevent bad data from entering systems. These include:
    - Code changes
    - go.mod dependency and min version changes
    - New features
    - Licensing of dependencies.
- Semantic versioning must be adhered to, for reasons above.
- Interactions with the community must be respectful and helpful.
- Ideally been working with and have good understanding of the Go language.

Thanks you in advance

Update: Additional Details

For those wanting to get involved please reply in a comment here and I can start adding people to the Triage team. Once I can see a few contributions, reviews and approvals of PR's then will be able to promote to full maintainers write permissions.

**Comment by nodivbyzero** [comment]
Hi Dean,

Thank you for your transparency and for all the time and effort you’ve dedicated to maintaining this package over the years!

I’d like to offer my help in maintaining this package. I’m confident I can contribute meaningfully and help ensure it continues to grow and serve the community effectively.

Please let me know how I can proceed to get involved and support the package.

**Comment by banjoh** [comment]
Thanks for all the great work you've done @deankarn

**Comment by annaelizabeth2019** [comment]
I would be interested in working on this package! :D

**Comment by turisap** [comment]
I would like to participate in this project as well.

**Comment by chipaca** [comment]
Hi!

I've been using this package for a few years now, at work and in personal projects. I'd like to help with maintaining it if that's alright.

Thank you for everything,

**Comment by deankarn** [comment]
@chipaca @turisap @annaelizabeth2019 sorry for the delay, I’ve added you to the triage team.

**Reply by turisap** [reply]
so what is the best way to start out? just to pick up any issue or an MR and fix / review it?

**Comment by ardbp** [comment]
I have been using this package for last couple years now, I would also like to participate in maintaining this project if its appropriate. Thank you!

**Reply by nodivbyzero** [reply]
Hello,  
If you're interested in maintaining the project, the best way to start is by contributing actively. You can review pull requests, test new changes, provide feedback, submit your own PRs, improve documentation, and help answer questions in the project's discussion board. 
Encouraging others to contribute is also a great way to support the community.

**Reply by ardbp** [reply]
Sounds good Thank you!

**Comment by MadsRC** [comment]
@deankarn I'd love to help out!

**Comment by divyanshu-tiwari** [comment]
@deankarn I would love to contribute as well.

**Comment by bigzoo** [comment]
@deankarn, would love to help out!

**Comment by IamMayankThakur** [comment]
@deankarn As a long term user, I would love to contribute and give back to the project

**Comment by shishirsriv** [comment]
Hey Dean, 
thanks for putting in all the efforts. I am interested in supporting this package.
-Shishir

**Comment by bolusarz** [comment]
Hey Dean,
I would love to contribute to the wonderful work you are doing here

**Comment by gergely-pepper** [comment]
@deankarn @nodivbyzero please choose more people, also I'm ready to contribute. Eg. this PR looks like done and tested, conflicts resolved, but not merged for a long time now: https://github.com/go-playground/validator/pull/1284

**Reply by deankarn** [reply]
I've added a few more people to the Triage team to try and help with the load.

I want to add a few more people to the maintainers team to be able to merge, but of course would like to meet and vet who I give such permissions to. Unfortunately I just haven't had the time to do this yet, life's been busy lately, but hoping it should settle down around Dec. where I should be able to find some time for this.

Thanks everyone for your patience.

**Comment by RodrigoMattosoSilveira** [comment]
@RodrigoMattosoSilveira If you need additional collaborators, I'm interested.

**Comment by georgepsarakis** [comment]
@deankarn 👋 I'd be happy to contribute too.

**Comment by spa5k** [comment]
I'll be happy to contribute as well. @deankarn
